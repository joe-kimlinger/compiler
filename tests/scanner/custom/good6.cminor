// operators
5+2;
3++1;
4 ++ - 1;
8 --- 2;
8 * 7;
(6 ^ 3);
((7 / 1));
