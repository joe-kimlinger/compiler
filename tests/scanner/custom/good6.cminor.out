COMMENT
INTEGER_LITERAL 5
PLUS
INTEGER_LITERAL 2
SEMICOLON
INTEGER_LITERAL 3
INCREMENT
INTEGER_LITERAL 1
SEMICOLON
INTEGER_LITERAL 4
INCREMENT
MINUS
INTEGER_LITERAL 1
SEMICOLON
INTEGER_LITERAL 8
DECREMENT
MINUS
INTEGER_LITERAL 2
SEMICOLON
INTEGER_LITERAL 8
STAR
INTEGER_LITERAL 7
SEMICOLON
LPAREN
INTEGER_LITERAL 6
EXPONENT
INTEGER_LITERAL 3
RPAREN
SEMICOLON
LPAREN
LPAREN
INTEGER_LITERAL 7
SLASH
INTEGER_LITERAL 1
RPAREN
RPAREN
SEMICOLON
